mlir-sycl/test/Transforms/constant-propagation.mlir

```diff
@@ -1,39 +1,53 @@
-// RUN: sycl-mlir-opt %s -mlir-pass-statistics -split-input-file \
-// RUN: -sycl-constant-propagation 2>&1 | FileCheck %s
-
-// COM: First split
-
-// CHECK-LABEL: ConstantPropagationPass
-// CHECK-NEXT:   (S) 1 num-propagated-constants   - Number of propagated constants
-// CHECK-NEXT:   (S) 1 num-replaced-explicit-args - Number of replaced explicit arguments
-// CHECK-NEXT:   (S) 0 num-replaced-implicit-args - Number of replaced implicit arguments
-
-// COM: Second split
-
-// CHECK-LABEL: ConstantPropagationPass
-// CHECK-NEXT:   (S) 11 num-propagated-constants   - Number of propagated constants
-// CHECK-NEXT:   (S)  0 num-replaced-explicit-args - Number of replaced explicit arguments
-// CHECK-NEXT:   (S) 11 num-replaced-implicit-args - Number of replaced implicit arguments
+// RUN: sycl-mlir-opt %s -split-input-file -sycl-constant-propagation | FileCheck %s
 
 // COM: Check we can detect %c is constant
```

Comment:
Where is %c ? Expand the comment to describe what this test is testing?

Reply:
I changed the comment a bit. Think it's clearer now